app/src/main/res/layout/main_fragment.xml

```diff
@@ -43,9 +43,19 @@
         android:layout_width="wrap_content"
         android:layout_height="48dp"
         android:text="Tip Out"
-        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintBottom_toBottomOf="@id/settings_button"
         app:layout_constraintEnd_toEndOf="parent"
         app:layout_constraintStart_toStartOf="parent"
         app:layout_constraintTop_toBottomOf="@+id/count_in_button" />
 
+    <Button
```

Comment:
This could probably be a Toggle button if it just toggles night mode on and off.